air/tests/test_module/features/tetraplets/security_tetraplets.rs

```diff
@@ -16,6 +16,7 @@
 
 use air::SecurityTetraplet;
 use air_test_utils::prelude::*;
+use pretty_assertions::assert_eq;
```

Comment:
let's shuffle it

Reply:
How can we shuffle it? Both are `other imports from non-std` (not super/crate).

Reply:
It's good, I misread it.